extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -19,7 +19,8 @@ define('ext.wikia.adEngine.provider.directGpt', [
 		sraEnabled = !context.opts.disableSra,
 		atfSlots = [
 			'TOP_LEADERBOARD',
-			'GPT_FLUSH'
+			'GPT_FLUSH',
+			'TOP_RIGHT_BOXAD'
```

Comment:
It breaks non-SRA, isn't it?

Reply:
It does not. SRA added `TOP_RIGHT_BOXAD` to atfSlots. Now it does not since it's already there.